base/libgit2/walker.jl

```diff
@@ -1,5 +1,31 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitRevWalker(repo::GitRepo)
+
+A `GitRevWalker` *walks* through the *revisions* (i.e. commits) of
+a git repository `repo`. It is a collection of the commits
+in the repository, and supports iteration and calls to [`map`](@ref)
+and [`count`](@ref) (for instance, `count` could be used to determine
+what percentage of commits in a repository were made by a certain
+author).
+
+# Examples
+```julia
+oids = LibGit2.with(LibGit2.GitRevWalker(repo)) do walker
+    LibGit2.map((oid,repo)->string(oid), walker, by = LibGit2.Consts.SORT_TIME)
```

Comment:
You didn't use spaces around the `=` in `by` in the example below this one. It would be good to make them consistent, and in this case IMO it looks nicer without the spaces as it is in the second example.